bootstrap/pom.xml

```diff
@@ -15,8 +15,20 @@
 			<groupId>com.google.inject</groupId>
 			<artifactId>guice</artifactId>
 			<version>${guice.version}</version>
+            <exclusions>
```

Comment:
is the exlusion really needed if the version is specified in the pom file as well?